apps/iir_blur/iir_blur_generator.cpp

```diff
@@ -155,6 +155,14 @@ class IirBlur : public Generator<IirBlur> {
         // Scheduling is done inside blur_cols_transpose.
         output = blur;
 
+        // Hardcode all CPU/GPU kernel bounds via Halide's constant bound propagation.
+        input.dim(0).set_bounds(0, 1536).set_stride(1);
+        input.dim(1).set_bounds(0, 2560).set_stride(1536);
+        input.dim(2).set_bounds(0, 3).set_stride(1536 * 2560);
+        output.dim(0).set_bounds(0, 1536).set_stride(1);
+        output.dim(1).set_bounds(0, 2560).set_stride(1536);
+        output.dim(2).set_bounds(0, 3).set_stride(1536 * 2560);
+
```

Comment:
Re: OSX Metal buffer allocation error.

Hardcoding the input and output image sizes in the `iir_blur` pipeline appears to resolve the Metal GPU buffer size limit error.